corehq/apps/hqwebapp/views.py

```diff
@@ -613,13 +613,35 @@ def jserror(request):
             browser_version = parsed_agent['browser'].get('version', TAG_UNKNOWN)
             browser_name = parsed_agent['browser'].get('name', TAG_UNKNOWN)
 
+    url = request.POST.get('page', None)
+    domain = '_unknown'
+    if url:
+        parts = url.split('/')
```

Comment:
Would `get_domain_from_url` work here?

Reply:
Ah, that's a good point. `get_domain_from_url` (which I didn't know about before) appears to take as input the path that comes after the <protocol>://<domain> section of the url (i.e. `/a/{domain}/...`, whereas `url` here includes the protocol and domain part of the url. So I can't pass it directly in, but I can pass it through a real URL parser to get the path, and then pass the path to this function. That's better than my makeshift parsing here